### ISSUE TITLE
setting audio quality "best" doesnt actually change the quality best

### ISSUE THREAD

**Issue by pokegamer5547** [libre-tube/LibreTube#6058]
### Steps to reproduce

Set the audio quality as best in settings

### Expected behavior

It should select the one with highest bitrate and quality

### Actual behavior

It doean't reapect that and always usea "mp4a.40.2 0kbps" as audio quality. Idk if it's due to HLS. 

### LibreTube version

0.23.2

### Android version

android 13 lineage os 20

### Other details

The download button does show higher quality options though. 
And the audio source option also doesn't seem to provide any additional options.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
HLS doesn't provide bitrate information apparently, only DASH does. So I don't think we can do anything about it for HLS.

**Comment by pokegamer5547** [comment]
> HLS doesn't provide bitrate information apparently, only DASH does. So I don't think we can do anything about it for HLS.

That's also true, but how does newpipe used to do it tho? They always said even before DASH got implemented that newpipe app always picked highest bitrate audio.

**Comment by pokegamer5547** [comment]
@Bnyro so i followed this comment instructions and disabled piped proxy and use hls and it's still working so i assume since i disabled use hls it's using dash. Yet the bitrate is still low.

**Comment by Bnyro** [comment]
Pinging team members is against the code of conduct, first and last warning.

**Comment by pokegamer5547** [comment]
> Pinging team members is against the code of conduct, first and last warning.

Ah i understand, i'm sorry about that :(

**Comment by Bnyro** [comment]
> disabled piped proxy and use hls and it's still working so i assume since i disabled use hls it's using dash
>
No, when disabling the Piped proxy HLS is enforced because Dash playback without proxy was broken by YouTube some time ago, see https://github.com/libre-tube/LibreTube/issues/5524.

**Comment by Bnyro** [comment]
see https://github.com/libre-tube/LibreTube/issues/7482
